### PR TITLE
Fixing errors with regexfile and chmod/chown [Alt]

### PR DESCRIPTION
**Pull request by PromoFaux**
Re-do of #676 to encompass requested changes and to base properly on `dev` branch

### CODE REVIEW

File: bash_functions.sh [hunk 1]
```diff
@@ -27,12 +27,13 @@ prepare_configs() {
     chown pihole:root /etc/lighttpd
     chown pihole:pihole "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" "/var/log/pihole" "${regexFile}"
```
Comment (MichaIng) [inline bash_functions.sh:28]: Consequently `"${regexFile}"` can then be removed above as well, to mute the last error message.

Comment (PromoFaux) [reply bash_functions.sh:28]: Thanks, well spotted. Deleting with half an eye...

Comment (PromoFaux) [reply bash_functions.sh]: Except now I have to wait for the rebuild again 🙄

**Comment by PromoFaux** [comment]
Tested locally - all still working

**Pull request merged**